substrafl/remote/register/register.py

```diff
@@ -85,51 +84,21 @@
 """
 
 
-def _copy_local_packages(
-    path: Path, local_dependencies: typing.List[Path], python_major_minor: str, operation_dir: Path
-):
+def iter_dockerfile_install_command(paths: typing.List[Path], python_major_minor: str) -> str:
+    for path in paths:
+        yield f"COPY {path}  {path} \nRUN python{python_major_minor} -m pip install --no-cache-dir {path} \n"
+
+
+def _copy_local_packages(path: Path, python_major_minor: str, operation_dir: Path, dependencies: Dependency):
     """Copy the local libraries given by the user and generate the installation command."""
-    dependencies_buffer = []
     path.mkdir(exist_ok=True)
-    for dependency_path in local_dependencies:
-        dest_path = path / dependency_path.name
-        if dependency_path.is_dir():
-            shutil.copytree(
-                dependency_path,
-                dest_path,
-                ignore=shutil.ignore_patterns(
-                    "local-worker",
-                    TMP_SUBSTRAFL_PREFIX + "*",
-                ),
-            )
-        elif dependency_path.is_file():
-            shutil.copy(dependency_path, dest_path)
-        else:
-            raise ValueError(f"Does not exist {dependency_path}")
-
-        dependencies_buffer.append(f"{dest_path.relative_to(operation_dir)}")
```

Comment:
This line is needed to have relative paths, and not absolute paths, in the Dockerfile